File: src/main/java/com/quanweng/shopping/pojo/UserTrace.java
package com.quanweng.shopping.pojo;

import lombok.Data;
import java.time.LocalDateTime;

@Data
public class UserTrace {
    private Long id;
    private String userId;      // 用户ID或NO_LOGIN
    private String ip;          // 访问IP
    private String region;      // 地区（预留）
    private String action;      // login/register/navigation
    private String actionData;  // 相关数据
    private String requestSessionID; // 请求唯一标识
    private LocalDateTime createTime;
}

File: src/main/java/com/quanweng/shopping/pojo/UserTrace.kt
package com.quanweng.shopping.pojo

import java.time.LocalDateTime

data class UserTrace(
    var id: Long? = null,
    var userId: String? = null,          // 用户ID或NO_LOGIN
    var ip: String? = null,              // 访问IP
    var region: String? = null,          // 地区（预留）
    var action: String? = null,          // login/register/navigation
    var actionData: String? = null,      // 相关数据
    var requestSessionID: String? = null,// 请求唯一标识
    var createTime: LocalDateTime? = null
)
